public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -384,6 +385,22 @@ export class LokiDatasource
     return res.data || [];
   }
 
+  async queryStatsRequest(query: LokiQuery): Promise<QueryStats> {
+    const labelMatcher = `{${query.expr.split('{')[1].split('}')[0]}}`;
```

Comment:
I think we should rather use something like https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/modifyQuery.ts#L142 to get the correct matcher. 

Btw, how do we use this endpoint for queries with multiple stream selectors? As "easiest" something like `rate({foo="bar"}[5m]) + rate({foo="baz"}[5m])`